### DISCUSSION TITLE
Error deploy streamlit.io

### DISCUSSION THREAD

**Discussion by ReynaldiGev** [teamalgoritma/community#360] [Q&A] [answered]
Gagal deploy di streamlit.io

![Screenshot (17)](https://user-images.githubusercontent.com/70183099/192756231-94ae82f8-3e82-4053-9225-9ea32ddf151f.png)

**Comment by idoali** [comment, marked as answer]
Selamat siang Pak Reynaldi

Bapak bisa coba tambahkan file baru bernama `packages.txt` lalu kemudian tuliskan `libtk8.6` di file itu. Lokasi penempatan dari file `packages.txt` ini ada di lokasi yang sama seperti `requirements.txt` dan `app.py`.

File `packages.txt` ini berguna untuk menambahkan dependencies berbasis Linux pada aplikasi bapak. Berikut beberapa referensi lebih lanjutnya pak :
1. https://discuss.streamlit.io/t/tkinter-import-failed-during-app-deployment/23529 
2. https://docs.streamlit.io/streamlit-cloud/get-started/deploy-an-app/app-dependencies

Semoga bisa membantu menyelesaikan masalahnya pak. Terima kasih
